integration/docker/csi/alluxio/nodeserver.go

```diff
@@ -258,6 +234,27 @@ func getAndCompleteFusePodObj(nodeId string, req *csi.NodeStageVolumeRequest) (*
 	return csiFusePodObj, nil
 }
 
+func (ns *nodeServer) createFusePodIfNotExist(fusePod *v1.Pod) error {
+	if _, err := ns.client.CoreV1().Pods(os.Getenv("NAMESPACE")).Create(fusePod); err != nil {
+		if strings.Contains(err.Error(), "already exists") {
+			glog.V(4).Infof("Fuse pod %s already exists. Skip creating pod.", fusePod.Name)
+		} else {
+			return status.Errorf(codes.Internal, "Failed to launch Fuse Pod at %v.\n%v", ns.nodeId, err.Error())
+		}
+	}
+	return nil
+}
+
+func checkIfMountPointReady(mountPoint string) error {
+	command := exec.Command("bash", "-c", fmt.Sprintf("cat /proc/mounts | grep %v | grep alluxio-fuse", mountPoint))
```

Comment:
Does it work for the two alluxio docker image?

Reply:
Do you mean the [alluxio/alluxio](https://hub.docker.com/r/alluxio/alluxio/) and [alluxio/alluxio-enterprise](https://hub.docker.com/r/alluxio/alluxio-enterprise/) docker images, or something else?

Reply:
I think she meant alluxio/alluixo and alluxio/alluxio-dev. Yes they both work